opdracht-2.js

```diff
@@ -0,0 +1,79 @@
+//Uitwerkingen opdracht 2
+
+//opdracht 2a: Hoeveel tv's zijn er al verkocht? Schrijf een script dat dit berekent. Log de uHoeveel tv's zijn er al verkocht? Schrijf een script dat dit berekent. Log de uitkomst in de console.
+
+//1. Maak een variabele aan waarin wordt opgeslagen hoeveel tv's er zijn verkocht;
+//2. Declareer een for-loop;
+//3. Tel de aantallen van inventory.sold bij elkaar op;
+//4. Console log de variabele.
+
+let tvSold = 0
+
+for (let i = 0; i < inventory.length; i++) {
+    tvSold = tvSold + inventory[i].sold
```

Comment:
Hier kun je de volgende operator gebruiken:
```suggestion
    tvSold += inventory[i].sold
```